scrapy/extensions/telnet.py

```diff
@@ -69,7 +69,7 @@ def start_listening(self):
                     extra={'crawler': self.crawler})
 
     def stop_listening(self):
-        self.port.stopListening()
+        return self.port.stopListening()
```

Comment:
This seems unrelated to this pull request.

Reply:
Again, without `return` raises ResourceNotReleased warning on pypy interpreter. It breaks some tests.